packages/backend/src/lib/backends/github.sync.ts

```diff
@@ -227,6 +272,8 @@ export const githubRepositorySync = async (
 
     await syncFiles(gitInstance, insight, previousInsight);
 
+    insight.contributors = await getInsightContributors(insight, yaml);
```

Comment:
Moved getting contributors here since we need to have loaded the YAML config beforehand.